packages/jest/migrations.json

```diff
@@ -14,6 +14,11 @@
       "version": "9.0.0-beta.1",
       "description": "Upgrades jest-preset-angular to 8.0.0",
       "factory": "./src/migrations/update-9-0-0/update-9-0-0"
+    },
+    "update-9.0.1": {
+      "version": "9.0.1-beta.1",
```

Comment:
@FrozenPandaz - should this still be targetting `9.0.0-beta.1` or was that only to solve the local problem I had?

Reply:
Yes, target `9.0.0-beta.1` because we still haven't released `9.0.0` and maybe just move your schematic in with mine? 🤷‍♂

Reply:
I actually did that initially - but I thought you wanted to release `9.0.0` soon, so I thought this could go in `9.0.1` - should I move everything back in your migration?